### PR TITLE
PS-6103 Document simple LDAP authentication plugin

### PR DESCRIPTION
**Pull request by patrickbirch**
updated index
added simple-ldap with examples

### CODE REVIEW

File: doc/source/security/simple-ldap.rst [hunk 1]
```diff
@@ -0,0 +1,217 @@
+.. _simple-ldap:
+
+======================================================
+Using Simple LDAP Authentication
+======================================================
+
+This feature is **Experimental** quality.
+
+This feature was implemented in |Percona Server| version :rn:`8.0.18-9`.
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst]: 8.0.19

File: doc/source/security/simple-ldap.rst [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+LDAP (Lightweight Directory Access Protocol) provides an alternative method to
+access existing directory servers, which maintain information about
+individuals, groups, and organizations.
+
+The Percona Simple LDAP plugin is a free and Open Source implementation of the
+|MySQL|'s Simple LDAP plugin.
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst]: MySQL enterprise

File: doc/source/security/simple-ldap.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    * - authentication_ldap_simple_bind_root_dn
+      - Root distinguished name (DN)
+      -
+      -
+      -
+      - global
+      - yes
+      - string
+    * - authentication_ldap_simple_bind_root_pwd
+      - Password for the root distinguished name
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst:57]: A current issue is that this password can be read by anybody who can access the server. We probably should mention this somewhere.

File: doc/source/security/simple-ldap.rst [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    * - authentication_ldap_simple_group_search_filter
+      - Custom group search filter
+      - (|(&(objectClass=posixGroup)(memberUid={UA}))(&(objectClass=group)(member={UD})))
+      -
+      -
+      - global
+      - yes
+      - string
+    * - authentication_ldap_simple_init_pool_size
+      - Initial size of teh connection pool to the LDAP server
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst]: the

File: doc/source/security/simple-ldap.rst [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+      -
+      -
+      - global
+      - yes
+      - string
+    * - authentication_ldap_simple_server_port
+      - LDAP server TCP/IP port number
+      - 389
+      - 1
+      - 32376
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst]: 65535

File: doc/source/security/simple-ldap.rst [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+      - uid
+      -
+      -
+      - global
+      - yes
+      - string
+
+
+For simple LDAP authentication, you must specify the authentication_ldap_simple
+plugin in the ``CREATE USER`` statement.
```
Comment (dutow) [inline doc/source/security/simple-ldap.rst]: or you can also `ALTER` existing users.

File: doc/source/security/simple-ldap.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst]: fixed.

File: doc/source/security/simple-ldap.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst]: updated.

File: doc/source/security/simple-ldap.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst:57]: added a warning.

File: doc/source/security/simple-ldap.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst]: updated.

File: doc/source/security/simple-ldap.rst (hunk 6 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst]: Added alter user

File: doc/source/security/simple-ldap.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/security/simple-ldap.rst]: fixed.

**Review by dutow — APPROVED** [review]

**Pull request merged**